Echo.json5

```diff
@@ -194,6 +194,7 @@
     { name: 'AREnableAuctionImprovementsSignals', value: true },
     { name: 'AREnableSubmitArtworkTier2Information', value: true },
     { name: 'AREnableCuratorsPicksAndInterestSignals', value: true },
+    { name: 'AREnableDynamicHomeView', value: false }, // aka Art Advisor Milestone 2
```

Comment:
Should I actually set this to `true`? That should be fine so long as https://github.com/artsy/eigen/pull/10886 is un-merged

(I may be being overly cautious here)

Reply:
I think it’s fine to set it to true now, as our goal is to gradually enroll users with the new app version. But it’s also fine to change it to true later - just let’s make sure we don’t forget about it.